go-controller/pkg/node/controllers/egressip/egressip.go

```diff
@@ -874,8 +921,16 @@ func (c *Controller) repairNode() error {
 			}
 			linkIdx := link.Attrs().Index
 			linkName := link.Attrs().Name
-			expectedIPRoutes.Insert(getDefaultRoute(linkIdx, isEIPV6).String())
-			expectedAddrs.Insert(getNetlinkAddress(eIPNet, linkIdx).String())
+			// copy routes associated with link to new route table
+			linkRoutes, err := generateRoutesForLink(link, isEIPV6)
+			if err != nil {
```

Comment:
so you are getting routes here for the link, to determine what is stale and should be removed later in the repair?

Reply:
yes